code/datums/uplink/announcements.dm

```diff
@@ -12,7 +12,7 @@
 /datum/uplink_item/abstract/announcements/fake_centcom/New()
 	..()
 	name = "[current_map.boss_name] Update Announcement"
-	item_cost = round(DEFAULT_TELECRYSTAL_AMOUNT / 2)
+	item_cost = 2
```

Comment:
That would allow the purchase of 12 Central Command announcements.
Use something like TC / 3 or a set it between 8-10.

Reply:
done